styles/style.css

```diff
@@ -3,3 +3,200 @@
   padding: 0;
   box-sizing: 0;
 }
+
```

Comment:
.aboutus-section class'ını ekleyip `max-width: 1920px` `width: 100%`  ve daha büyük ekranlarda sayfada ortalanması için  `margin-inline: auto` değerleri vermen gerkiyor